### DISCUSSION TITLE
【收集】面试专业名词

### DISCUSSION THREAD

**Discussion by jiangzhonglian** [apachecn/Interview#359] [Ideas]
业务建模：
数据建模：使用数据建模技术来分析数据对象，以此洞悉数据的内在涵义。

统计分析：
对比分析：
聚类分析：它是将相似的对象聚合在一起，每类相似的对象组合成一个聚类(也叫作簇)的过程。这种分析方法的目的在于分析数据间的差异和相似性。
回归分析：
判别分析：
相关性分析：
相关系数：是研究变量之间线性相关程度的量（较为常用的是皮尔森相关系数）

异常检测：在数据集中搜索与预期模式或行为不匹配的数据项。

数据采样：
数据增强：

特征选择：

特征工程：
数据清洗：对数据进行重新审查和校验的过程，目的在于删除重复信息、纠正存在的错误，并提供数据一致性。
降维：

**Comment by kangyuqi** [comment]
已收到您的邮件，谢谢。
